feature-tests/utils/dynamodb-update-revert.ts

```diff
@@ -0,0 +1,109 @@
+import { DynamoDB, UpdateItemCommand, UpdateItemCommandInput } from '@aws-sdk/client-dynamodb';
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
```

Comment:
Hey Liam - I found this link that explains what I mentioned briefly in dev techy: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/introduction/
You are importing the dynamoDB library in two different way here:
- modular sdk v3 way where you import only the basic client and the individual commands you need
- you are also importing `DynamoDB` which I think is an class representation of the client that under the hood will import all possible operations that you could do on DynanoDB, resulting in more code being imported.
From my understanding this will have an impact on performance and how large the build for the feature tests is, but in the grand scheme of things this is probably alright, not a massive issue :) just wanted to flag it. I would personally stick to the modular set up like we have in our DynamoDB service class, but as you said you were you trying something different

Reply:
this is a very good point, will probably change it to the dynamoDBClient rather than importing the other one to increase performance, didn't even consider that initially, thank you for adding this along with the link!